moc_sprint_tools/cmd/copy_cards.py

```diff
@@ -0,0 +1,44 @@
+import click
+import github
+import logging
+
+from moc_sprint_tools.sprintman import BoardNotFoundError
+from moc_sprint_tools import defaults
+
+LOG = logging.getLogger(__name__)
+
+
+def CommaDelimitedList(val):
+    if val is None:
+        return None
+
+    return val.split(',')
+
+
+@click.command(name='copy-cards')
+@click.option('-f', '--from', 'src', metavar='BOARD_NAME')
+@click.option('-t', '--to', 'dst', metavar='BOARD_NAME')
+@click.option('-c', '--copy-columns', type=CommaDelimitedList)
+@click.option('-i', '--ignore-columns', type=CommaDelimitedList)
+@click.option('-I', '--no-default-ignore', is_flag=True)
+@click.pass_context
+def main(ctx, src, dst, copy_columns, ignore_columns, no_default_ignore):
+    api = ctx.obj
+
+    if not (src and dst):
+        raise click.ClickException('you must provide both a source and destination board')
+
+    if ignore_columns is None:
+        if no_default_ignore:
```

Comment:
why isn't this an `and`?
```python
if ignore_columns is None and no_default_ignore:
    ignore_columns = []
```

Reply:
I think because at one point there was additional logic there...but there's not now, so you're right, there's no good reason for that. Good catch!